runtime/vm/createramclass.cpp

```diff
@@ -3397,7 +3381,6 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 			 *              + J9ClassEnsureHashed (inherited)
 			 *             + J9ClassHasOffloadAllowSubtasksNatives
 			 *            + J9ClassIsPrimitiveValueType | J9ClassAllowsInitialDefaultValue
-			 *           + J9ClassAllowsNonAtomicCreation
```

Comment:
Replace this with Unused instead of removing the line like is done  in line 3387.